app/metrics.yaml

```diff
@@ -23,3 +23,152 @@ distribution:
     notification_emails:
       - dmu@mozilla.com
     expires: "2020-05-01"
+pages:
+  page_load:
+    type: timing_distribution
+    lifetime: application
```

Comment:
Are you sure this is really an `application` lifetime metric? This sounds like a `ping` lifetime metric: you really want the metric to be cleared after the ping is sent, not when the application is shut down.

Reply:
Note that the lifetime consideration also applies to all the other metrics below.